core/controllers/reader.py

```diff
@@ -137,18 +104,13 @@ def get(self, exploration_id):
         if (feconf.EXPLORATION_URL_EMBED_PREFIX in self.request.uri or
                 self.request.get('iframed')):
             self.values['iframed'] = True
-        try:
-            # If the exploration does not exist, a 404 error is raised.
-            exploration_data_values = _get_exploration_player_data(
-                exploration_id, version, collection_id, can_edit)
-        except Exception:
+
+        if not _exploration_exists(exploration_id, version, collection_id):
             raise self.PageNotFoundException
 
-        self.values.update(exploration_data_values)
         self.values['iframed'] = True
```

Comment:
Is this value actually used?

If not, what actually handles the "iframed" parameter?

Reply:
Then is it worth setting it as a class property instead, i.e. self.iframed = True? If we don't need self.values at all for page templates we should try to not use it at all, to reduce confusion.

Reply:
Done.